apps/fmr/views/request-sent.html

```diff
@@ -0,0 +1,14 @@
+{{<partials-page}}
+  {{$pageTitle}}
+    {{#t}}pages.request-sent.confirmation{{/t}} &ndash; {{#t}}journey.serviceName{{/t}} &ndash; GOV.UK
+  {{/pageTitle}}
+  {{$page-content}}
+    <div class="govuk-panel govuk-panel--confirmation" role="alert">
+     <h1 class="govuk-panel__title">
+        {{#t}}pages.request-sent.confirmation{{/t}}
+      </h1>
+    </div>
```

Comment:
I guess the rest of this is added with the reference generation ticket?

Reply:
yes, that will be covered separately